sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -2736,7 +2736,7 @@ lsc_atomic_update(T *p, Toffset offset, __ESIMD_NS::simd_mask<N> pred = 1) {
 /// @tparam Op is operation type.
 /// @tparam T is element type.
 /// @tparam N is the number of channels (platform dependent).
-/// @tparam DS is the data size.
+/// @tparam DS is the data size (unused/obsolete).
```

Comment:
The old lsc_atomic_update() can pass u16u32 and get result vector if dwords as the result, right?
If Yes, then we cannot say is is unused/obsolete.
It is true though that DS is not used for _new_ API.

Reply:
yeah youre right, fixed